pkg/result/filter.go

```diff
@@ -210,14 +211,39 @@ func getIgnoredIDs(ignoreFile string) []string {
 		if strings.HasPrefix(line, "#") || line == "" {
 			continue
 		}
-		ignoredIDs = append(ignoredIDs, line)
+		// Process all fields
+		fields := strings.Fields(line)
+		if len(fields) > 1 {
+			exp, err := getExpirationDate(fields)
+			if err != nil {
+				log.Logger.Warnf("Error while parsing expiration date in .trivyignore file: %s\n", err)
+				continue
+			}
+			if !exp.IsZero() {
+				now := time.Now()
+				if exp.Before(now) {
+					continue
+				}
+			}
+		}
+		ignoredIDs = append(ignoredIDs, fields[0])
 	}
 
 	log.Logger.Debugf("These IDs will be ignored: %q", ignoredIDs)
 
 	return ignoredIDs
 }
 
+func getExpirationDate(fields []string) (time.Time, error) {
+	for _, field := range fields {
+		if strings.HasPrefix(field, "exp:") {
+			return time.Parse(time.RFC3339, strings.TrimPrefix(field, "exp:"))
```

Comment:
I guess most users don't want to specify hour/minute/second. What if we define a layout like `"2006-01-02"`?

Reply:
Yeah, that absolutely makes sense 👍 
I changed it to `2006-01-02` and updated the tests, docs and PR description.